sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ConnectionStringProperties.java

```diff
@@ -21,10 +21,10 @@
     private static final String ENTITY_PATH = "EntityPath";
     private static final String ERROR_MESSAGE_FORMAT = "Could not parse 'connectionString'. Expected format: "
         + "'Endpoint={endpoint};SharedAccessKeyName={sharedAccessKeyName};"
-        + "SharedAccessKey={sharedAccessKey};EntityPath={eventHubName}'. Actual: %s";
+        + "SharedAccessKey={sharedAccessKey};EntityPath={entityPath}'. Actual: %s";
 
     private final URI endpoint;
-    private final String eventHubName;
+    private final String entityPath;
```

Comment:
I thought we wanted to change from path to name. Are we going back to using entity path?

Reply:
In AMQP, this is the entity path, when we surface it in our client API, it is EventHubName